docs/services/shippingservice.md

```diff
@@ -152,6 +152,21 @@ Adding a span event:
     info!("Tracking ID Created: {}", tid);
 ```
 
+### Reqwest HTTP client instrumentation
```

Comment:
Just a nit pick 😅 

Could you move that to the following section?
https://github.com/styblope/opentelemetry-demo/blob/shippingservice-client-span/docs/services/shippingservice.md#adding-grpchttp-instrumentation

Maybe we could even split the section in 2:
`Adding gRPC instrumentation` with what is there.

and: `Adding HTTP instrumentation` with what you have added.

Reply:
yup makes sense, 1 sec